scripts/concatenate_root_certs.py

```diff
@@ -0,0 +1,93 @@
+#
```

Comment:
It seems that this script has to be used manually to update the bundle when any of cert files are changed, right? Maybe GHA can do that?

Reply:
The problem is that the change needs to be committed - so we would need to have workflow also commit and push to the repo.